pages/blog.md

```diff
@@ -0,0 +1,1465 @@
+                    >Stephen Ferguson</a> wrote:</div>
+            <div class="body"><b>Library Name: </b>New York Society Library<br/><b>Location:
+                </b>New York, NY<br/><br/>Now online --
+                See<br/><br/>http://www.nysoclib.org/ledger/index.html<br/><br/>The New York
+                Society Library's earliest surviving charging ledger, which records borrowing
+                activity during the period between July 1789 and April 1792.<br/><br/></div>
+
+        </div>
+        <div class="comment">
+            <div class="header">On 2010-11-12 19:39:43 <a href="mailto:ferguson@princeton.edu"
+                    >Stephen Ferguson</a> wrote:</div>
+            <div class="body"><b>Library Name: </b>Soldiers Free Library<br/><b>Location:
+                </b>Washington, DC<br/><br/>The Soldiers Free Library was founded on October 15,
+                1862 in Washington, D.C. by Elida Rumsey (1841-1919) and her future husband John
+                Allen Fowle. The library opened with 1,500 books and 800 magazines. Both Rumsey and
+                Fowle were actively involved in relief work for Union soldiers in the Washington,
+                D.C. area. .....<br/>More
+                at<br/>http://libraryhistorybuff.blogspot.com/2010/11/soldiers-free-library.html</div>
+
+        </div>
+        <div class="comment">
+            <div class="header">On 2010-08-27 21:23:03 <a href="mailto:stustiff@verizon.net">stuart
+                    stiffler</a> wrote:</div>
+            <div class="body"><b>Library Name: </b>Painesville Library Aociation
+                    Library<br/><b>Location: </b>Painesville, Lake Co., Ohio<br/><br/> Had 25
+                periodical subscriptions in Spring of 1844; offered books for auction<br/> in late
+                1844.<br/> Painesville Telegraph, Feb. 18, 1841; May 5, 1841; March 1, 1841.</div>
+
+        </div>
+        <div class="comment">
+            <div class="header">On 2010-08-27 21:14:46 <a href="mailto:stustiff@verizon.net">stuart
+                    stiffler</a> wrote:</div>
+            <div class="body"><b>Library Name: </b>Painesville public library<br/><b>Location:
+                </b>painesville lake co, ohio<br/><br/>A social library; apparently operated less
+                than a year when books were sold at auction" <br/> Painesville Telegraph Sept. 1,
+                1824</div>
+
+        </div>
+        <div class="comment">
+            <div class="header">On 2010-08-27 21:05:02 <a href="mailto:stustiff@verizon.net">stuart
+                    stiffler</a> wrote:</div>
+            <div class="body"><b>Library Name: </b>Nelson Library Company<br/><b>Location:
+                </b>Nelson, Portage County, Ohio<br/><br/> In re founding date:1829----library was
+                established
+                "probaBLY<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/> IN
```

Comment:
There are a dozen or so places with several `<br/>` tags in a row creating pretty big gaps in the page. Maybe trim everything down to at most 2 `<br/>`'s in a row?